src/wiki/keyboard-shortcuts.md

```diff
@@ -206,3 +206,18 @@ This article will introduce you to the essential shortcuts that will help you na
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>→</kbd>  | Switches between virtual desktops to the right. |
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>←</kbd>  | Switches between virtual desktops to the left.  |
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>F4</kbd> | Closes the current virtual desktop.             |
+
+## How to find them easily with PowerToys
+PowerToys is a great tool that every Windows should have, and it’s developed by Microsoft, that has very powerful tools that will make you more productive.
```

Comment:
Tbf, the header "How to find them easily with PowerToys" isn't relevant to the wiki page. You will not find every shortcuts mentioned in PowerToys.

Reply:
It only will show the commands with win key

Reply:
This is getting really complicated 😵‍💫